DigitalOffice/Tests/TimeManagementServiceTests/Validators/CreateWorkTimeRequestValidatorTests.cs

```diff
@@ -0,0 +1,251 @@
+using FluentValidation;
+using FluentValidation.TestHelper;
+using Moq;
+using NUnit.Framework;
+using System;
+using System.Collections.Generic;
+using LT.DigitalOffice.TimeManagementService.Database.Entities;
+using LT.DigitalOffice.TimeManagementService.Models;
+using LT.DigitalOffice.TimeManagementService.Repositories.Filters;
+using LT.DigitalOffice.TimeManagementService.Repositories.Interfaces;
+using LT.DigitalOffice.TimeManagementService.Validators;
+
+namespace LT.DigitalOffice.TimeManagementServiceUnitTests.Validators
+{
+    public class CreateWorkTimeRequestValidatorTests
+    {
+        private Mock<IWorkTimeRepository> repositoryMock;
+
+        private IValidator<CreateWorkTimeRequest> validator;
+
+        private readonly Guid WorkerUserId = Guid.NewGuid();
+        private readonly DateTime Date = DateTime.Now;
+        private readonly TimeSpan Time = new TimeSpan(6, 0, 0);
+        private readonly string Title = "Worked...";
+        private readonly Guid ProjectId = Guid.NewGuid();
+        private readonly string Description = "Did something";
+
+        [SetUp]
+        public void Setup()
+        {
+            repositoryMock = new Mock<IWorkTimeRepository>();
+
+            repositoryMock.Setup(x => x.GetUserWorkTimes(It.IsAny<Guid>(), It.IsAny<WorkTimeFilter>()))
+                .Returns(new List<DbWorkTime>());
+
+            validator = new CreateWorkTimeRequestValidator(repositoryMock.Object);
+        }
+
+        [Test]
+        public void SuccessfulWorkTimeValidation()
+        {
+            var request = new CreateWorkTimeRequest
+            {
+                WorkerUserId = WorkerUserId,
+                Date = Date,
+                Time = Time,
+                Title = Title,
+                ProjectId = ProjectId,
+                Description = Description
+            };
+
+            var result = validator.Validate(request);
+            Assert.IsTrue(result.IsValid);
+        }
+
+        #region WorkerUserId
+        [Test]
+        public void FailValidationEmptyWorkerUserId()
+        {
+            var request = new CreateWorkTimeRequest
+            {
+                //WorkerUserId = WorkerUserId,
```

Comment:
Very interesting decision. Pls, edit. WorkerUserId = Guid.Empty will be better

Reply:
Fixed.